.github/workflows/ci.yml

```diff
@@ -23,12 +22,11 @@ jobs:
       TURBO_TEAM: ${{ vars.TURBO_TEAM }}
     steps:
       - uses: actions/checkout@v4
-      - uses: pnpm/action-setup@v2
+      - uses: pnpm/action-setup@v4
```

Comment:
v4 automatically supports `packageManager` in `package.json`, so the corepack command is no longer necessary